libraries/tomcat_users_conf.rb

```diff
@@ -30,11 +30,11 @@ class TomcatUsers
   has_many :users, User, tag: 'user'
 end
 
-require 'pry'; binding.pry
+# require 'pry'; binding.pry
 
-test = TomcatUsers.parse('/Users/aaronl/tomcat-users.xml')
+# test = TomcatUsers.parse('/Users/aaronl/tomcat-users.xml')
 
-puts 'hi'
+# puts 'hi'
 
 # class TomcatUsersConf < Inspec.resource(1)
 #   name 'tomcat_users_conf'
```

Comment:
Generally commented out code should just be removed, it is in the git history if a user needs it, but overall it just clutters the repo.

Reply:
I'm ok with removing the debugging stuff, but I'm reluctant to remove the class that was defined there and got commented out since that class definition seems to be the reason why this file was made.